src/realtime/index.ts

```diff
@@ -7,7 +7,7 @@ import { PresencePayload } from './presence/presence-handler'
 type MqttProtocoldIdOptions = 'MQTT'
 type IMessagePayloadTypes = 'json' | 'string'
 
-export interface RealtimeMessage {
+export interface RealtimeMessage<ExtraPayload = never> {
```

Comment:
is the never type intended here?

Reply:
yes it is

The result for this would be that extraPayload would be never only in case the getExtraPayload function was not passed

Check following screenshots for example

<img width="1016" alt="Screenshot 2023-02-02 at 10 31 01" src="https://user-images.githubusercontent.com/6594267/216272136-a41409be-ea9c-4495-b628-3d002a9da7c1.png">
<img width="1016" alt="Screenshot 2023-02-02 at 10 31 21" src="https://user-images.githubusercontent.com/6594267/216272161-56e86be2-cfcc-42aa-ab60-2de6bff2066c.png">
<img width="1016" alt="Screenshot 2023-02-02 at 10 31 28" src="https://user-images.githubusercontent.com/6594267/216272171-ddc9349f-d331-4d6e-a071-6a6129f503b9.png">

Reply:
Alright thanks for the explenation!